src/DataViewer.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component, ReactChildren } from "react";
+import * as Types from "./types";
+
+type Cell = {
+  component?: Component<{
+    row: number;
+    column: number;
+    value: ReactChildren;
+  }>;
+};
+
+type Props = Types.ICellComponentProps<Cell, Node> & {
+  formulaParser: any; // FormulaParser for hot-formula-parser
+};
+
+const toView = (value: boolean | string | Node) => {
+  if (typeof value === "boolean") {
+    return <div className='boolean'>{value ? "TRUE" : "FALSE"}</div>;
+  }
+
+  return value;
+};
+
+const DataViewer: React.FC<Props> = ({
+  getValue,
+  cell,
+  column,
+  row,
+  formulaParser
+}: Props) => {
+  const rawValue = getValue({ data: cell, column, row }) as
+    | boolean
+    | string
+    | Node;
```

Comment:
Why the `as` here? Considering it's an already typed prop above, doesn't fully make sense.

Reply:
This is the `getValue()`type: https://github.com/FlatFilers/react-spreadsheet/pull/3/files/1c368355c138c392ef4979122e72740f3871822a#diff-41df78d86ed4de80aae9b0b9560feb2bR47